packages/api/src/record/controller.test.ts

```diff
@@ -319,3 +324,96 @@ describe("record", () => {
     expect(logger.error).toHaveBeenCalledWith(testError);
   });
 });
+
+describe("patch record", () => {
```

Comment:
We should have a test case covering the scenario where the update query fails at the database.